src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo:React.FC<Props> = ({ user }) => {
+  return (
+    <>
+      <a className="UserInfo" href={`mailto:${user.email}`}>
+        {user.name}
+      </a>
+    </>
```

Comment:
Useless extra fragment here 
```suggestion
      <a className="UserInfo" href={`mailto:${user.email}`}>
        {user.name}
      </a>
```